### DISCUSSION TITLE
Can websites with lots of channels be listed here?

### DISCUSSION THREAD

**Discussion by jkemp814** [iptv-org/community#263] [Show and tell]
I was searching for a source for a channel that wasn't working and sound a site where it was working. I noticed they have lots of channels listed. Can someone take a look and see if those channels can be added here? I would create ADD: requests if they can. Seems to be 344 listed.

https://canlitv.com/tr

**Comment by BellezaEmporium** [comment]
Sounds good to me. 

As long as it stays here (like I did with the Astra discussion, see below), it should be completely fine.

For this number of channels, I suggest you put the names on a Ghostbin or Pastebin, and that you send the link in it. Would be painful to look at them all.
